packages/frint/src/App.ts

```diff
@@ -0,0 +1,418 @@
+import { Container, createContainer, Provider, resolveContainer } from 'frint-di';
+import find = require('lodash/find');
+import findIndex = require('lodash/findIndex');
```

Comment:
is this a TypeScript specific syntax? I am used to seeing imports like this:

```js
import findIndex from 'lodash/findIndex'; 

// OR

import * as findIndex from 'lodash/findIndex';
```

Reply:
Yes, these don't work in TypeScript, I found this alternative way here: https://github.com/lodash/lodash/issues/3192

If I do
```
import omit from 'lodash/omit';
```
that gives me this error: `[ts] Module '"frint/node_modules/@types/lodash/omit"' has no default export.`  
On the other hand, it works with `rxjs`, I think because they do exports differently.  
Rxjs does (in `find.d.ts`)
```
export declare function find...
```
And `lodash` is doing this (in `omit.d.ts`)
```
import { omit } from "./index";
export = omit;
```

I'm not really familiar with what the difference is between these two styles, is it familiar to you?